layouts/partials/home/options.html

```diff
@@ -103,28 +100,53 @@ <h5>Modules</h5>
             </ul>
         </div>
 
-        <div class="home-options__option" style="transform: translateY(-4.5px);">
+        <div class="home-options__option" style="min-height:394px;min-width:328px">
             <div class="tile-opener">
-            <a href="/platforms">
+            <a href="/rdi">
                 <div class="home-options__option__icon">
-                    <img src="./theme-flex/img/ico-platforms.svg" alt="Platforms Icon" class="platforms-icon">
+                    <img src="./theme-flex/img/ico-connect.svg" alt="Redis Data Integration Icon" style="height: 70%; margin-top:20px;">
                 </div>
-                <h5>Kubernetes</h5>
-                <p>Redis Enterprise deployed on containerized software platforms and integrated with our partners.</p>
+                <h5>Redis Data Integration</h5>
+                <p>Redis Data Integration imports existing data into Redis databases.</p>
             </a>
             </div>
             <hr>
             <ul>
                 <section>
-                    <li><i class="fa fa-angle-right fa-lg category-icon"></i><a
-                            href="/kubernetes/architecture">Architecture</a></li>
-                    <li><i class="fa fa-angle-right fa-lg category-icon"></i><a
-                            href="/kubernetes/rec/">Clusters</a></li>
+                    <li><i class="fa fa-angle-right fa-lg category-icon"></i><a href="/rdi">Quick Start</a></li>
+                    <li><i class="fa fa-angle-right fa-lg category-icon"></i><a href="/rdi/Architecture">Architecture</a></li>
```

Comment:
The architecture link on the home page leads to a 404.
```suggestion
                    <li><i class="fa fa-angle-right fa-lg category-icon"></i><a href="/rdi/architecture">Architecture</a></li>
```